### ISSUE TITLE
 Can't resolve '@angular/router.ngfactory

### ISSUE THREAD

**Issue by redoc123** [ngx-rocket/generator-ngx-rocket#359]
## I'm submitting a...

- [x ] Bug report  
- [ ] Feature request
- [ ] Documentation issue or request

## Current behavior

I am getting the following exception when i run. 
./src/app/app.module.ngfactory.js
Module not found: Error: Can't resolve '@angular/router.ngfactory' in '/src/app'

## Expected behavior

There should nt be error.

## Minimal reproduction of the problem with instructions

==Remove all traces of QuoteService from the generated applications

modified:   src/app/home/home.component.html
	modified:   src/app/home/home.component.spec.ts
	modified:   src/app/home/home.component.ts
	modified:   src/app/home/home.module.ts
	deleted:    src/app/home/quote.service.spec.ts
	deleted:    src/app/home/quote.service.ts

## Environment

```
- generator version: 5.0.1
- node version: v8.9.4
- npm version: 5.6.0
- OS:  Mac OSX
```

Others:

**Comment by sinedied** [comment]
Duplicate of #352 , see https://github.com/ngx-rocket/generator-ngx-rocket/issues/352#issuecomment-412026999 and https://github.com/ngx-rocket/generator-ngx-rocket/issues/352#issuecomment-412033257.

**TL;DR**: Latest `@angular/cli` release is buggy, remove the `--aot` flag from the `start` script in your `package.json` script as a temporary workaround 😞

**Comment by artuska** [comment]
Issue still exists.

```
Angular CLI: 7.3.2
Node: 10.0.0
OS: darwin x64
Angular: 7.2.5
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.2
@angular-devkit/build-angular     0.13.2
@angular-devkit/build-optimizer   0.13.2
@angular-devkit/build-webpack     0.13.2
@angular-devkit/core              7.3.2
@angular-devkit/schematics        7.3.2
@angular/cli                      7.3.2
@ngtools/webpack                  7.3.2
@schematics/angular               7.3.2
@schematics/update                0.13.2
rxjs                              6.4.0
typescript                        3.2.4
webpack                           4.29.0
```

**Comment by ImATechnophile** [comment]
Issue fixed:
Just update your cli and core to the latest version.
 ng update @angular/cli
 ng update @angular/core
